app/src/main/resources/templates/thymeleaf/views/experiment-page/experiment-page_content.html

```diff
@@ -0,0 +1,32 @@
+<div th:replace="~{views/experiment-page/experiment-description}"></div>
+
+<link rel="stylesheet" th:href="@{/resources/css/experiment.css}">
+<link rel="stylesheet" th:href="@{/resources/css/alt-customized-bootstrap-3.3.5.css}">
+
+<div class="row expanded margin-top-large">
+    <div class="small-12 columns">
+        <div id="experiment-page" ></div>
+    </div>
+</div>
+
+<div id="feedback-button">
+</div>
+
+<link rel="stylesheet" th:href="@{/resources/css/react-popup.css}" type="text/css" media="all">
+<script defer th:src="@{/resources/js-bundles/feedbackForm.bundle.js}"></script>
+
+<script defer th:src="@{/resources/js-bundles/experimentPage.bundle.js}"></script>
+
+<script th:inline="javascript">
+  document.addEventListener("DOMContentLoaded", function() {
+    experimentPage.render({
+      atlasUrl: '[(@{/})]',
+      content: /*[(${experimentContent})]*/ {},
+      target: 'experiment-page'
+    });
+    feedbackForm.render({
+      feedbackFormLink: 'https://www.ebi.ac.uk/support/gxa',
```

Comment:
What happens if the app is deployed to wwwdev or www-test?

Reply:
@amnonkhen I understand what is your thought behind your question. You think it should be not a hardcoded value and the base path of the URL should change depending on which environment we are on. Normally, I would agree with you, but this time we are talking about a link that takes to an EBI supported page (not our team).
Sadly that page has not got www-test equivalent at all, there is a wwwdev page but it is saying : Debug mode! - Messages submitted via this form will NOT be processed, please use [www.ebi.ac.uk/about/contact/support/gxa](https://www.ebi.ac.uk/about/contact/support/gxa) if you have landed on this page by mistake.
Also, for obvious reason there is no localhost equivalent of this URL.

In short, I would leave this URL hardcoded.